packages/sdk/server-ai/src/LDAIClientImpl.ts

```diff
@@ -13,6 +13,7 @@ import { LDClientMin } from './LDClientMin';
 interface LDMeta {
   variationKey: string;
   enabled: boolean;
+  version: number;
```

Comment:
```suggestion
  version?: number;
```
Unless we are migrating all the versions.